node/sc/bridge_accounts.go

```diff
@@ -37,6 +37,11 @@ import (
 
 const (
 	DefaultBridgeTxGasLimit = 5000000
+
+	ParentOperatorStr                  = "parentOperator"
+	ChildOperatorStr                   = "childOperator"
+	DEFAULT_PARENT_BRIDGE_ACCOUNT_NAME = "parent_bridge_account"
```

Comment:
@hyunsooda Is there any reason that this variable is declared in another form unlike other variables?

Reply:
Sorry, I missed that one. Changed with the following commit.